camviewer_ui_impl.py

```diff
@@ -694,7 +706,28 @@ def __init__(
             self.onCameraSelect(int(cameraIndex))
         except Exception:
             pass
+
+        # Set the right hand area's width based on font sizes
+        font = self.ui.labelCamera.font()
+        fm = QFontMetricsF(font)
+        charwidth = fm.averageCharWidth()
+        # We want to make it wide enough for 50 average chars
+        width = 50 * charwidth
```

Comment:
Did this come from any specific calculation?  Or was it just an estimation?  (seems good either way)

Reply:
The fixed width predating all of this camviewer work was 300 and it looked good on my monitor at 300. My monitor's charwidth was fractions of a pixel over 6, and 50*6=300, so I picked 50 to match my existing sizing and hoped that this ratio would carry forward to other displays.